src/utils/NebulaKeyUtils.cpp

```diff
@@ -193,6 +194,9 @@ std::vector<std::string> NebulaKeyUtils::snapshotPrefix(PartitionID partId) {
     } else {
         result.emplace_back(vertexPrefix(partId));
         result.emplace_back(edgePrefix(partId));
+        result.emplace_back(IndexKeyUtils::indexPrefix(partId));
```

Comment:
It will affect balance?

Reply:
We didn't move index data :bug:. I don't even know what will happen before.